releases/mainnet/1.10.1/contracts.json

```diff
@@ -0,0 +1,296 @@
+{
```

Comment:
Why is the no `mainnet` 1.10.0 to compare against? The diff between this and 1.9.9 is non-trivial.

Reply:
Oh nevermind, just did see it because of lexical ordering >:c